scripts/2500/NIPT/syncnipt.bash

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# script to rsync a run to the NIPT server
+
+shopt -s nullglob
+set -eu
+
+VERSION=5.4.2
+echo "Version $VERSION"
+
+##########
+# PARAMS #
+##########
+
+RUNBASE=/home/proj/production/flowcells/2500/runs/
+NIPTBASE=/home/proj/production/flowcells/2500/nipt/
+NIPTOUTPATH=/srv/nipt_runs/
+MAILTO_ERR=clinical-demux@scilifelab.se
+
+#############
+# FUNCTIONS #
+#############
+
+failed() {
+    echo "Fail to sync ${RUN}" | mail -s "ERROR syncing NIPT $(hostname):${RUN}" ${MAILTO_ERR}
+}
+trap failed ERR
+
+#######
+# RUN #
+#######
+
+for RUN in ${RUNBASE}/*; do
+  RUN=$(basename ${RUN})
+  NOW=$(date +"%Y%m%d%H%M%S")
+  if [[ ! -e ${NIPTBASE}${RUN} ]]; then
+    # simple NIPT detection
+    if grep -qs Description,cfDNAHiSeqv1.0 ${RUNBASE}${RUN}/SampleSheet.csv; then
+      if [[ ! -e ${RUNBASE}${RUN}/SampleSheet.ori ]]; then
+        cp ${RUNBASE}${RUN}/SampleSheet.csv ${RUNBASE}${RUN}/SampleSheet.ori
+      fi
+
+      # transform SampleSheet from Mac/Windows to Unix
+      if grep -qs $'\r' ${RUNBASE}${RUN}/SampleSheet.csv; then
+          sed -i 's/
+/\n/g' ${RUNBASE}${RUN}/SampleSheet.csv
```

Comment:
Yes.

This is what it looks like for me:
![image](https://user-images.githubusercontent.com/26384108/109656641-eee71600-7b64-11eb-9d12-9fc020f3bf67.png)

Guess what ^M means? :)

I've pushed it to this branch like that, installed it on stage and it works. But when I initially cloned the branch it looks like it does above and it did not work because the `sed` was now replacing `spaces` with `\n` instead of `^M` with `\n`. I'll see if I can fix this.

Reply:
Resolved, see below.

Reply:
Aha! Wierd but all good then!